pkg/aws/gomock_reflect_3295326204/prog.go

```diff
@@ -1,64 +0,0 @@
-package main
```

Comment:
Could you add these more `gomock_reflect_` in the gitignore in passing? 

https://github.com/aws/aws-application-networking-k8s/blob/3f89fd8c39c66f54af349eb9823fc8baf1a5944c/.gitignore#L18

Reply:
This shouldn't exist at all, so it shouldn't go into .gitignore, it was an artifact from a previous build issue that got checked in